DocumentFormat.OpenXml.Tests/FlatOpcTests.cs

```diff
@@ -145,6 +147,32 @@ public void CanCreateFlatOpcPresentationDocuments8()
             }
         }
 
+        [Fact(Skip = "#380")]
+        public void CanCreateFlatOpcPresentationDocumentsDynamically()
+        {
+            using (MemoryStream stream = new MemoryStream())
+            using (PresentationDocument presentation = PresentationDocument.Create(stream, PresentationDocumentType.Presentation))
+            {
+                try
```

Comment:
Can you remove the try/catch. It's okay if these tests fail (in fact they should) since we have the skip attribute on them. It's not expected that they throw exceptions